### DISCUSSION TITLE
Not Working on MacOS

### DISCUSSION THREAD

**Discussion by sly-fox-git** [yermak/AudioBookConverter#583] [MacOS]
cd /Applications/AudiobookConverter.app

./Contents/MacOS/AudiobookConverter

Error Result: AudiobookConverter” is damaged and can’t be opened. You should move it to the Trash.

Same error as launching the app as normal.

macOS Monterey Version 12.6

**Comment by ac4lt** [comment]
Seeing the same issue on MacOS 13.0.1.

**Comment by letternews** [comment]
[patrickg75](https://github.com/patrickg75)
[on Feb 28, 2022](https://github.com/yermak/AudioBookConverter/issues/455#discussioncomment-2266296)
In the installation wiki, there's how to start it via terminal.
https://github.com/yermak/AudioBookConverter/wiki/Installation#macos-x

Make sure, that in SystemPreferences > Security & Privacy > Developer Tools > Terminal is checked.

But still adding files doesn't work

**Comment by sly-fox-git** [comment]
Indeed. Adding files doesn't work

**Comment by jessearmstrong** [comment]
Something must have changed because these instructions do not work for me.

**Comment by yermak** [comment]
As a heads-up, I'm working on pure native version for Mac to overcome all those problems (I faced many others), as I tired to fight with Mac specific issues in current implementation - I spent all might time on mac specific issues.
Mac version will be launched very soon, it will be very basic at the begining, but hopefully all those issues will be not relevant.

**Comment by jessearmstrong** [comment]
That sounds amazing! Looking forward to it.

**Comment by yermak** [comment]
Duplicates #580

**Comment by yermak** [comment]
Beta for Mac OS X is available
https://github.com/yermak/AudioBookConverter/issues/580#issuecomment-2268738667
